build/android/makeArchive7z.sh

```diff
@@ -0,0 +1,2 @@
+
+ tar cf - $1 -P | pv -s $(du -sb $1  | awk '{print $1}') | 7z a -si -m0=lzma2 -mx=3 $2.tar.7z
```

Comment:
these is a serious security issue here for ci because args.gn (which is in the output dir) contains sensitive information like api keys and must be excluded

Reply:
removed the sensitive info from the archive